### PR TITLE
Dtspo-3229: JD: use azure keyvault to store secrets

### PR DESCRIPTION
**Pull request by mokainos**
Dtspo-3229: JD: mount secrets from az keyvault

### CODE REVIEW

File: charts/jd-bureau/values.yaml [hunk 1]
```diff
@@ -11,4 +11,13 @@ nodejs:
     SMARTSURVEY_EXPORT_V2: "Dashboard Export"
   ingressSessionAffinity:
     enabled: true
-    sessionCookieName: affinity
\ No newline at end of file
+    sessionCookieName: affinity
+  keyVaults:
+    juror-digital-vault-{{ .Values.global.environment }}
```
Comment (timja) [inline charts/jd-bureau/values.yaml]: environment should be appended automatically

Comment (mokainos) [reply charts/jd-bureau/values.yaml]: shall i leave the name as "juror-digital-vault"

Comment (timja) [reply charts/jd-bureau/values.yaml]: yeah it's in their code, best to live with that :p

File: charts/jd-bureau/values.yaml [hunk 2]
```diff
@@ -11,4 +11,13 @@ nodejs:
     SMARTSURVEY_EXPORT_V2: "Dashboard Export"
   ingressSessionAffinity:
     enabled: true
-    sessionCookieName: affinity
\ No newline at end of file
+    sessionCookieName: affinity
+  keyVaults:
+    juror-digital-vault
```
Comment (timja) [inline charts/jd-bureau/values.yaml]:
```suggestion
    juror-digital-vault:
```

File: charts/jd-public/values.yaml [hunk 3]
```diff
@@ -6,4 +6,12 @@ nodejs:
     NODE_ENV: development
   ingressSessionAffinity:
     enabled: true
-    sessionCookieName: affinity
\ No newline at end of file
+    sessionCookieName: affinity
+  keyVaults:
+    juror-digital-vault
```
Comment (timja) [inline charts/jd-public/values.yaml]:
```suggestion
    juror-digital-vault:
```

**Review by timja — APPROVED** [review]

**Pull request merged**